docs/api-docs/apps/guide/apps-04-devloper-portal.md

```diff
@@ -1,87 +1,77 @@
 # Managing Apps in the Developer Portal
 
-
-
-App developers create, edit, and submit apps for approval using the [Developer Portal](https://devtools.bigcommerce.com/). In [Beginning App Development](/api-docs/apps/guide/development) we briefly touched on how to create a draft app. In this article, we'll go over how to perform other common app management tasks in the [Developer Portal](https://devtools.bigcommerce.com/). For detailed instructions on submitting apps for approval, see [Publishing Apps](/api-docs/apps/guide/publishing).
-
-## Signing in
-
-Sign in or create a [Developer Portal](https://devtools.bigcommerce.com) account at [devtools.bigcommerce.com](https://devtools.bigcommerce.com).
+Create, edit, and submit apps for approval using the [Developer Portal](https://devtools.bigcommerce.com/). In [Beginning App Development](/api-docs/apps/guide/development) we briefly touched on how to create a draft app. In this article, we'll go over how to perform other common app management tasks. To get started, sign in or create an account in the [Developer Portal](https://devtools.bigcommerce.com/).  
 
 <!-- theme: info -->
-> #### Note
-> * `DRAFT` apps can only be installed on stores owned by the same email address as the developer portal account's email address.
+> #### Store email address constraint
+> Apps that aren't approved for distribution through the [Apps Marketplace](https://bigcommerce.com/apps) can only be installed on stores owned by the same email address as the developer portal account's email address.

```

Comment:
```suggestion
> You can only install unapproved apps that are not for distribution through the [Apps Marketplace](https://bigcommerce.com/apps) on stores owned by the same email address as the developer portal account's email address.

```

Reply:
decline. this is a grey area. some apps are custom to the store and BC is fine with those being `private apps` or even `draft apps` if they're in live beta. thus, I don't want to refer to non-marketplace apps as `unapproved`, but I haven't yet introduced the draft/private app distinction.